gc_little_helper_II.user.js

```diff
@@ -6014,6 +6014,7 @@ var mainGC = function() {
                     // so we have to load informations for all caches.
                     $('#gmCacheInfo .map-item').each(function () {
                         gccode = $(this).find('.code').html();
+                        $(this).find('dl dt')[0].innerHTML = $(this).find('dl dt')[0].innerHTML.replace(/Created by:/,"by:").replace(/Erstellt von:/,"von:");
```

Comment:
Der Text "Created by" ist Sprachabhängig. Willst du die anderen Sprachversionen (zum Beispiel Deutsch) auch anpassen oder belassen wir es beim Englischen?

Reply:
Da wir den GClh ja nur in englisch anbieten ...
Deutsch hab ich nur wegen dir mitgeliefert, weil du es ja in deutsch nutzt. :)
Ich weiß auch nicht wie mans richtig sprachabhängig macht, das soll ja vielleicht mal kommen, hörte ich, auch wenn ich eine Übersetzung für Zeitverschwendung halte und etwas Werbung im englischen Sprachraum für sinnvoller halten würde. 

Ich könnte die Bezeichnung komplett weglassen und sie als title so wie vorhanden, also sprachabhängig, hinter den Usernamen hängen. Was meinst du?

Reply:
Es war nur eine Frage/Anregung. 
Ich nutze seit dem letzten Problem das wir mit der Sprache hatten GC auch nur noch auf Englisch 😃.
Ich würde es auch so lassen. Eine "universelle" Sprachabhängigkeit wird denke ich nicht klappen, da der Text ja immer unterschiedlich ist und man auch in der jeweiligen Sprache unterscheiden müsste, was es bedeutet. Dafür ist das "Feature" zu klein.